### ISSUE TITLE
region administration

### ISSUE THREAD

**Issue by TanyaStere42** [WildMeOrg/codex-frontend#620]
## Context
Some regions are useful for hierarchical organization, but not useful for assignment and matching. Admins should be able to restrict these regions from being used.

## User Story
As admin on the region management page (/src/pages/fieldManagement/settings/defaultFieldComponents/Editors.jsx), I can set a binary option that indicates if a region can be selectable in the submission/sighting metadata edit form.

## Technical Notes
Dependent on https://github.com/WildMeOrg/houston/issues/944
Needs to be represented in region selection (submission page and sighting metadata edit)

## Design
![Image](https://github.com/WildMeOrg/codex-frontend/assets/62726866/df79b8e4-e48f-403c-950b-462f2303a8a9)

Text to replace Lorem Ipsum:
> Deselect any region that you want to present in the hierarchy but not use in direct matching comparisons (ex: the region is too big).

**Comment by naknomum** [comment]
there is now a [PR for the backend side](https://github.com/WildMeOrg/houston/issues/944) of this.  it adds support for a new boolean property on the individual region json: `placeholderOnly`, for example:
```
{
    "id": "4c837a69-7398-49db-b166-3d30b05b85c0",
    "name": "Fubar",
    "placeholderOnly": true
}
```
the new code will **cause a 400 HoustonError** under two circumstances, upon modifying the settings data via the api:
1. if `placeholderOnly: true` is set on a region location ID _which is in use as data by codex_
2. if a region location ID _which is in use as data_ **is removed**

**Comment by TanyaStere42** [comment]
closed with merge of #627
